### ISSUE TITLE
Update comment about how secret key is calculated

### ISSUE THREAD

**Issue by sshock** [rails/rails#35717]
This comment on line 412 of `railties/lib/rails/application.rb` needs to be updated:

```
# In test and development, this is simply derived as a MD5 hash of the application's name.
```

Since it changed in this commit: 4c74358

**Comment by sshock** [comment]
@tenderlove @eileencodes @jhawthorn

**Comment by jhawthorn** [comment]
Thank you! Fixed by d0334b8f017eb63b37359c738860046fb4daa710
